guides/release/getting-started/quick-start.md

```diff
@@ -31,10 +31,11 @@ you will have access to a new `ember` command in your terminal.
 You can use the `ember new` command to create a new application.
 
 ```bash
-ember new ember-quickstart
+ember new ember-quickstart --lang en
```

Comment:
Can we update the example language to `en-US`, to better match the one in the [ember-new-lang RFC](https://github.com/emberjs/rfcs/blob/master/text/0635-ember-new-lang.md)? I think this may also help distinguish the locale from `en-UK`, for example.

```suggestion
ember new ember-quickstart --lang en-US
```

Reply:
Actually, the RFC does mention the example of just `--lang en`. I think what you have currently is okay. 👍🏼